libs/RobotKit/include/RobotController.h

```diff
@@ -222,7 +222,9 @@ class RobotController : public interface::RobotController
 	}
 
   private:
-	std::chrono::seconds _sleep_timeout_duration {300};
+	system::robot::sm::logger logger {};
+
+	std::chrono::seconds _sleep_timeout_duration {10};
```

Comment:
Previously set at 300 seconds before enter sleeping state, seems not relevant to modify it with this PR

```suggestion
	std::chrono::seconds _sleep_timeout_duration {300};
```

Reply:
yup! sorry about that, corrected!